### PR TITLE
Add pending case and skip test case

### PR DESCRIPTION
**Pull request by z6wdc**
1. Add a pending case in `gorilla-reporter.js`
2. Add a skip test case in `test/basic.js`

### CODE REVIEW

File: test/basic.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ describe('Basic', () => {
     }, 1000);
   });
 
+  it('should only test in the correct environment', function(done) {
+      this.skip();
+      done();
```
Comment (dwi2) [inline test/basic.js]: Since it is not an asynchronous test, there is no need to call `done()` here.

File: test/basic.js [hunk 2]
```diff
@@ -15,6 +15,11 @@ describe('Basic', () => {
     }, 1000);
   });
 
+  it('should only test in the correct environment', function(done) {
+      this.skip();
```
Comment (dwi2) [inline test/basic.js:19]: if you call `skip()` on `it` instance, you'll be able to use arrow function here.
```js
it.skip(`this test`, () => {
...
});
```

File: test/basic.js (hunk 1 above)
Comment (z6wdc) [reply test/basic.js]: a6605c8c8c2071bf25b46ec83602b2932c6360e7
delete `done()`

File: test/basic.js (hunk 2 above)
Comment (z6wdc) [reply test/basic.js:19]: Yes, but sometimes we need to skip at runtime.
Like the following code,
```
  if (/* check test environment */) {
    // make assertions
  } else {
    this.skip();
  }
```

So I am not using arrow function here.

File: test/basic.js [hunk 3]
```diff
@@ -15,6 +15,10 @@ describe('Basic', () => {
     }, 1000);
   });
 
+  it('should only test in the correct environment', function(done) {
```
Comment (dwi2) [inline test/basic.js]: this `done` could be removed too.

**Review by dwi2 — APPROVED** [review]
LGTM with only one additional comment

File: test/basic.js (hunk 3 above)
Comment (z6wdc) [reply test/basic.js]: b5e7da9506907f8c417a4cf2fc20f9275aaac5c9
I have removed it.

**Pull request merged**